commit f148d75b0ba881512b367ec1a9b0586f25550b23
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-31T10:37:50-04:00 (10 weeks ago)
    
    TRITON-1845 grafana should use node v6
    TRITON-1846 update grafana base image to triton-origin-x86_64-18.4.0
    Reviewed by: Isaac Davis <isaac.davis@joyent.com>
    Approved by: Isaac Davis <isaac.davis@joyent.com>

diff --git a/Makefile b/Makefile
index 328ae32..2e78185 100644
--- a/Makefile
+++ b/Makefile
@@ -11,12 +11,11 @@
 NAME = grafana
 
 GO_PREBUILT_VERSION = 1.11.1
-NODE_PREBUILT_VERSION = v6.15.1
+NODE_PREBUILT_VERSION = v6.17.0
 ifeq ($(shell uname -s),SunOS)
     # We use a 64-bit node because grafana will not build with 32-bit node 6
     NODE_PREBUILT_TAG=zone64
-    # Allow building on other than triton-origin-multiarch-18.1.0
-    NODE_PREBUILT_IMAGE=1ad363ec-3b83-11e8-8521-2f68a4a34d5d
+    NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 ENGBLD_USE_BUILDIMAGE = true
@@ -33,10 +32,10 @@ ifeq ($(shell uname -s),SunOS)
     include ./deps/eng/tools/mk/Makefile.agent_prebuilt.defs
 endif
 
-# triton-origin-multiarch-18.1.0
-BASE_IMAGE_UUID = b6ea7cb4-6b90-48c0-99e7-1d34c2895248
+#  triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = $(NAME)
-BUILDIMAGE_PKGSRC = pcre-8.42 bind-9.10.7
+BUILDIMAGE_PKGSRC = pcre-8.42 bind-9.11.8
 BUILDIMAGE_DESC = SDC Grafana
 AGENTS = amon config registrar
 
diff --git a/deps/eng b/deps/eng
index ed10150..63e056b 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit ed10150657b1f3ae4f061c751daf365ed16ca8d5
+Subproject commit 63e056ba42fd426b9314e99d1cd92773f4dcdda9
